### PR TITLE
add support on Loongnix-Server (loong64)

### PR DESCRIPTION
**Pull request by ysicing**
add support on Loongnix-Server (loong64)

Signed-off-by: ysicing <i@ysicing.me>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -2,6 +2,8 @@ module tailscale.com
 
 go 1.19
 
+replace github.com/u-root/uio v0.0.0-20220204230159-dac05f7d2cb4 => github.com/loongarch64/uio v0.0.0-20220922081955-9136972a7648
```
Comment (bradfitz) [inline go.mod:5]: We don't like checking in replace directives. They're too invasive for people who depend on this repo.

Can you upstream your changes to `github.com/u-root/uio` first?

**Review by bradfitz — CHANGES_REQUESTED** [review]
(LGTM after removing the `replace` directive)

File: util/endian/little.go [hunk 2]
```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-//go:build 386 || amd64 || arm || arm64 || mips64le || mipsle || ppc64le || riscv64 || wasm
+//go:build 386 || amd64 || arm || arm64 || mips64le || mipsle || ppc64le || riscv64 || wasm || loong64
```
Comment (bradfitz) [inline util/endian/little.go:5]: This part I've merged for you in https://github.com/tailscale/tailscale/pull/6236

File: go.mod (hunk 1 above)
Comment (ysicing) [reply go.mod:5]: https://github.com/u-root/uio/pull/6 
Wait for this to be merged

**Comment by bradfitz** [comment]
Once #6703 is in, tailscaled builds fine. Then the go.mod and go.sum changes can be removed from this PR. Rebase?

**Review by bradfitz — APPROVED** [review]

**Pull request merged**